www/js/menu.js

```diff
@@ -329,6 +330,23 @@ module.exports = {
 
   showNotificationsModal,
 
+  showDisplayTab(fromMainMenu) {
+    search.activateNavLink('display', true);
+    search.activateNavPage('session', { id: 'display', label: 'Display' });
+
+    const isPresenterView = document.body.classList.contains('presenter-view');
+    let settingsClickSource = isPresenterView ? 'from_presenter_view' : 'not_from_presenter_view';
+    if (fromMainMenu) settingsClickSource += '_main_menu';
```

Comment:
i don't think the scenario should appended into one string. try to separate it out so the analytics can be filtered on better. @maneetpaul can provide some input on what would be easiest to report on.
```
 ------------------------------------------
   * eventCategory  | text |  yes     | Typically the object that was interacted with (e.g. 'Video')
   * eventAction    | text |  yes     | The type of interaction (e.g. 'play')
   * eventLabel     | text |  no      | Useful for categorizing events (e.g. 'Fall Campaign')
   * eventValue     | int  |  no      | A numeric value associated with the event (e.g. 42)
```

Reply:
I am assuming this is for the analytics tag for the settings button. What I need to know: 

- How often the Settings button in the top left pane of Presenter View is clicked
- How often people click the settings button via the File > Preferences menu